Airports/EGPI/SMR/Labels.txt

```diff
@@ -0,0 +1,5 @@
+"2" N055.40.58.148 W006.15.02.278 standHold
+"1" N055.40.59.908 W006.15.01.124 standHold
+"B1" N055.41.02.598 W006.14.57.425 standHold
+"B3" N055.41.04.850 W006.14.57.822 standHold
+"A1" N055.41.00.055 W006.15.19.901 standHold
```

Comment:
```suggestion
;Heli Parking
"1" N055.40.59.908 W006.15.01.124 standHold
"2" N055.40.58.148 W006.15.02.278 standHold

;Hold Points
"A1" N055.41.00.055 W006.15.19.901 standHold
"B1" N055.41.02.598 W006.14.57.425 standHold
"B3" N055.41.04.850 W006.14.57.822 standHold
```
Alphanumeric ordering is best, and having them split into titled sections. Not so much of an issue here but when doing more complex airports helps us maintain them.